Nodejs/Product/Nodejs/Snippets/1033/Nodejs/require.snippet

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>require</Title>
+      <Description>Code Snippet for requiring a module</Description>
+      <Shortcut>require</Shortcut>
+      <Author>Microsoft Corporation</Author>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Declarations>
+        <Literal>
+          <ID>myModule</ID>
+          <Default>myModule</Default>
+        </Literal>
+      </Declarations>
+      <Code Language="Nodejs">
+        <![CDATA[var $myModule$ = require('$myModule$');]]>

```

Comment:
make sure the cursor ends up at the end of the require statement in this snippet.